### PR TITLE
BZ1193484: Roles added to Projects are ignored by Business Central

### PR DESCRIPTION
**Pull request by manstis**
See https://bugzilla.redhat.com/show_bug.cgi?id=1193484

This commit adds the last piece of the puzzle; filtering for Social Events.

### CODE REVIEW

**Comment by manstis** [comment]
@ederign Please review before I cherry-pick for 6.3.x. I'd appreciate your opinion of the tests as I don't know much about the different types of Social Event. Thanks, Mike

**Comment by manstis** [comment]
@psiroky this looks an interesting failure?

**Comment by psiroky** [comment]
@manstis yeah, it failed to clone the source from Github. I've seen this several times over the past few days, but I am not sure what is the root cause. If it's on our network, or github is blocking the connections...

**Comment by manstis** [comment]
ok to test.

File: kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/pom.xml [hunk 1]
```diff
@@ -93,15 +103,15 @@
     </dependency>
 
     <!-- EJB for AsyncDispatcher -->
- <!--    <dependency>
```
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/pom.xml]: Delete it?

**Comment by ederign** [comment]
+1 .

**Comment by csadilek** [comment]
There's already a commit on master with the same title but with different content. So, -1, no idea what to do.

**Comment by manstis** [comment]
@csadilek Please note the commits have different messages.

**Comment by manstis** [comment]
@csadilek I modified the commit message.

**Comment by csadilek** [comment]
@manstis noted ;). Missed that it had a subsequent line.

**Comment by csadilek** [comment]
Thanks.

master: 9900c16bbfe22a1b3b4b6597a8d1e953a2afa344

**Comment by manstis** [comment]
@csadilek Yep, it wasn't ideal that part of the "fix" for this BZ (a) spanned Eder's changes for Social Event filtering, (b) was time constrained by the Product 6.2.x release. I'd have liked to keep the whole fix in a single commit.

**Pull request closed without merging**